src/devices/DCMotorController/README.md

```diff
@@ -0,0 +1,29 @@
+# DC Motor Controller
+
+## Summary
+
+This is a generic class to control any DC motor.
```

Comment:
Will it really work for any DC Motor? should we provide a list of devices where we have tested the binding and ensured that it works?

Reply:
any motor connected to the H-bridge yes except the stepper motors which work completely differently but uniform across all stepper motors